api/app/Policies/UserPolicy.php

```diff
@@ -227,6 +227,9 @@ protected function individualAbleToCheck(User $actor, string $roleId)
                 return $actor->isAbleTo('assign-any-role');
             case 'platform_admin':
                 return $actor->isAbleTo('update-any-platformAdminMembership ') || $actor->isAbleTo('assign-any-role');
+            case 'manager':
+                return $actor->isAbleTo('update-any-managerMembership ') || $actor->isAbleTo('assign-any-role');
+
```

Comment:
Does this mean managers can update other manager memberships?

Reply:
It shouldn't!  :laughing:   It means that someone is able to add or remove the manager relationship if they have the update-any-managerMembership permission or the 'assign-any-role' permission.  That's configured in rolepermission.php

Reply:
Ahh okay now it makes sense :+1: :laughing: